lib/uptime_web/components/layouts/app.html.heex

```diff
@@ -1,2 +1,12 @@
-<.flash_group flash={@flash} />
-<%= @inner_content %>
+<header class="px-6 lg:px-10 py-8">
```

Comment:
Not sure how I feel about layouts in a liveview app. This method feels more verbose since to change a few elements it requires duplicated html.